src/lib/format.ts

```diff
@@ -1,7 +1,7 @@
 const locale = 'en-US' // TODO: user preference
 
 const formatMoney = (
-  amount: string,
+  amount: string = '0',
```

Comment:
I'm not sure if 0 is really a good fallback value. If `amount` is `undefined` here, that smells more like an error in the application. And instead of communicating this to the user, we just put 0? I don't think that's a good solution. We should handle all errors before we call this method and then just assume `amount` to be a valid value here.

Reply:
Makes sense, yes. I'll remove it and check in the calling function.